index/index.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 Arduino SA
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package index
+
+import (
+	"bytes"
+	"encoding/hex"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"path"
+	"time"
+
+	"github.com/arduino/go-paths-helper"
+	"golang.org/x/crypto/openpgp"
```

Comment:
@umbynos Since the [x/crypto/openpgp](https://pkg.go.dev/golang.org/x/crypto/openpgp) is considered deprecated, it is required to change the library as done in the PNX-4200 ticket

Reply:
Already done in https://github.com/arduino/arduino-create-agent/pull/839